task5-import-to-s3/task.md

```diff
@@ -0,0 +1,60 @@
+# Task 5 (AWS Simple Storage Service - S3)
+
+**After Lecture 5 "Amazon Simple Storage Service
+(Amazon S3)"**
+
+## Prerequisites
+---
+
+- The task is a continuation of Homework 4 and should be done in the same repos
+- Install the latest version of AWS SDK (https://aws.amazon.com/ru/sdk-for-node-js/)
+- Install the CSV parser package (https://www.npmjs.com/package/csv-parser)
+- Sign in into AWS Console and create a new S3 bucket with a folder, called uploaded
+- Pull latest changes from nodejs-aws-fe (https://github.com/rolling-scopes-school/nodejs-aws-fe)
+
+## TASK 5.1
+---
+
+Create a new service, called **import-service**, with a its own **serverless.yml** file.
+
+Create a lambda function in that **serverless.yml** file, called **importProductsFile**, which will be triggered by the HTTP **GET** method.
+
+The requested URL should be **/import**.
+
+The file name will be passed in a **query string** as a **name** parameter and should be described in the **serverless.yml** file as a **request parameter**.
+
+Update **serverless.yml** with policies to allow lambda functions to interact with **S3**.
+
+A new **Signed URL** should be created with the following **Key: `uploaded/${fileName}`**.
+
+The response from the lambda should be the created **Signed URL**.
+
+The lambda endpoint should be integrated with the frontend by updating **import** property of the API paths configuration.
```

Comment:
Please provide link to FE line or module

Reply:
Done, added path to the file